src/e2e/src/browser-manager.ts

```diff
@@ -0,0 +1,32 @@
+import { MainPage } from './pages/main.page';
+import { browser, ProtractorBrowser } from 'protractor'
+
+
+let activeBrowser: ProtractorBrowser
+
+export async function getBrowser() {
+  if (!activeBrowser) {
+    console.log('Preparing browser')
+    activeBrowser = await prepareBrowser()
+  }
+
+  return activeBrowser
+}
+
+async function prepareBrowser(): Promise<ProtractorBrowser> {
+  // We are testing a React project. We need to disable Angular sync.
+  await browser.waitForAngularEnabled(false)
+
+  console.info('Getting page: ', browser.baseUrl);
+  await browser.get(browser.baseUrl, 10000)
+
+  // Wait for application to be up and connected to services
+  await waitForConnected(browser)
```

Comment:
Here is a change. Starting the test session now waits for the application to be connected to the broker.